web/src/context/installerL10n.tsx

```diff
@@ -230,6 +230,30 @@ function InstallerL10nProvider({ children }: { children?: React.ReactNode }) {
         reload(newLanguage);
       } else {
         setLanguage(newLanguage);
+
+        // load the translations dynamically, first try the language + territory
+        const po = newLanguage.replace("-", "_");
```

Comment:
If possible, I would move this logic to a function so the caller is easy to read.